### PR TITLE
Changed readme

### PR DESCRIPTION
**Pull request by Vijay-Nirmal**

### CODE REVIEW

**Comment by AakashSasikumar** [comment]
maas bro

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # BlindVison
-A fun project that uses stereo vision.
+A fun python project that uses stereo vision.
+
+# Roadmap
```
Comment (AakashSasikumar) [inline README.md]: Looks good

Comment (Vijay-Nirmal) [reply README.md]: I will change it

File: README.md [hunk 2]
```diff
@@ -1,4 +1,2 @@
 # BlindVison
-A fun python project that uses stereo vision.
-
-# Roadmap
+A fun python project that uses stereo vision
```
Comment (AakashSasikumar) [inline README.md]: cool

**Pull request merged**